frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa.obcTest.scenario.spec.ts

```diff
@@ -0,0 +1,284 @@
+import {
```

Comment:
Lets rename this file too `obc-test.spec.ts`.
Scenarios are used in protractor tests.